.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: Continuous Integration
+
+on: push
+
+jobs:
+  build_test:
+    name: Build and Run Jest Tests
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+      - name: Install Dependencies
+        run: npm ci
+      - name: Build
+        run: npm run build
+      - name: Run Tests
+        run: npm run test:coverage -- --runInBand
+  cypress:
+    runs-on: ubuntu-16.04
```

Comment:
Is there a reason for the different ubuntu version specifications? If so, perhaps a comment is in order.

Reply:
Most cypress examples used this version of ubuntu but there was one example that used latest. I just switched it to latest and it seems to work fine, so I'll go with that for consistency.